android/festago/app/src/main/java/com/festago/festago/data/repository/TicketDefaultRepository.kt

```diff
@@ -1,34 +1,20 @@
 package com.festago.festago.data.repository
 
 import com.festago.festago.data.service.TicketRetrofitService
-import com.festago.festago.domain.model.Stage
 import com.festago.festago.domain.model.Ticket
 import com.festago.festago.domain.model.TicketCode
-import com.festago.festago.domain.model.TicketState
 import com.festago.festago.domain.repository.TicketRepository
-import java.time.LocalDateTime
 
 class TicketDefaultRepository(
     private val ticketRetrofitService: TicketRetrofitService,
 ) : TicketRepository {
 
-    private val fakeTicket = List(10) {
-        Ticket(
-            id = it + 1L,
-            number = it + 100,
-            LocalDateTime.of(2023, 7, 30, 15, 0),
-            TicketState.BEFORE_ENTRY,
-            Stage(1, "테코대학교 축제 DAY1", LocalDateTime.of(2023, 7, 30, 18, 0)),
-            "",
-        )
-    }
-
     override suspend fun loadTicket(ticketId: Long): Ticket {
         return ticketRetrofitService.getTicket(ticketId).body()!!.toDomain()
     }
 
     override suspend fun loadTickets(): Result<List<Ticket>> {
-        return Result.success(fakeTicket)
+        return Result.success(ticketRetrofitService.getTickets().body()!!.toDomain())
```

Comment:
예외 간단하게 처리하는 게 어떨까요?
```suggestion
        val response = ticketRetrofitService.getTickets()
        return if (response.isSuccessful && response.body() != null) {
            Result.success(response.body()!!.toDomain())
        } else {
            Result.failure(Throwable("code: ${response.code()}, body: ${response.errorBody()?.toString()}}"))
        }
```

Reply:
좋아요 한번 적용해보겠습니다!